crates/control/src/role_assignment.rs

```diff
@@ -292,6 +295,7 @@ fn process_role_state_machine(
     game_controller_state: Option<&GameControllerState>,
     player_number: PlayerNumber,
     striker_trusts_team_ball: Duration,
+    optional_roles: &Vec<Role>,
```

Comment:
```suggestion
    optional_roles: &[Role],
```
a slice is more generic than a reference to a `Vec`

Reply:
done